binplace.targets

```diff
@@ -36,6 +36,7 @@
   <Target Name="GetTargetingPackDir">
     <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('NETCoreApp')) Or $(NuGetTargetMoniker.Contains('NETStandard')) Or $(NuGetTargetMoniker.Contains('UAP'))">
       <TargetingPackDir Include="$(RefPath)" />
+      <TargetingPackDir Condition="'$(TargetGroup)'!='$(_bc_TargetGroup)'" Include="$(RefRootPath)$(TargetGroup)/" />
```

Comment:
>Include="$(RefRootPath)$(TargetGroup) [](start = 75, length = 37)

this looks fine. just want to mention we have the initialization in dir.props

    <RefPath>$(RefRootPath)$(_bc_TargetGroup)/</RefPath>
    <NetStandardRefPath>$(RefRootPath)netstandard/</NetStandardRefPath>
    <NetFxRefPath>$(RefRootPath)netfx/</NetFxRefPath>

are we going to use these properties for anything?